package/samplers/smac_sampler/sampler.py

```diff
@@ -344,5 +353,5 @@ def _convert_to_config_space_design_space(
             else:
                 raise NotImplementedError(f"Unknown Hyperparameter Type: {type(distribution)}")
             if hp is not None:
-                config_space.add_hyperparameter(hp)
+                config_space.add(hp)
```

Comment:
`config_space.add_hyperparameter` is deprecated and it is recommended to use `add`.